src/layouts/login.vue

```diff
@@ -1,6 +1,166 @@
-<template>
+<!-- eslint-disable @intlify/vue-i18n/no-raw-text -->
+<template style="display: block; justify-content: center">
+  <div class="logo-container">
+    <img
+      alt="Ocial Name Logo"
+      class="mt-15 mb-8"
+      src="@/assets/images/Texto_Ocial.png"
+      style="width:70%; height: auto;" />
+  </div>
+
+  <div class="login-page">
+    <div class="form">
+      <form class="register-form">
+        <input
+          type="text"
+          placeholder="nombre" />
+        <input
+          type="text"
+          placeholder="correo electrónico" />
+        <input
+          type="password"
+          placeholder="contraseña" />
+        <input
+          type="password"
+          placeholder="confirmar contraseña" />
+        <button>create</button>
+        <p class="message">
+          ¿Ya tienes una cuenta? <a href="#">Inicia sesión</a>
+        </p>
+      </form>
+      <form class="login-form">
+        <input
+          type="text"
+          placeholder="correo electrónico" />
+        <input
+          type="password"
+          placeholder="contraseña" />
+        <button>iniciar sesión</button>
+        <p
+          class="message"
+          style="margin: 15px 0 0;">
+          ¿Aún no tienes cuenta? <a href="#">Regístrate</a>
+        </p>
+      </form>
+    </div>
+  </div>
+
+  <div class="container3">
+    <img
+      alt="Ocial Logo"
+      src="@/assets/images/Logo_Ocial.png"
+      style="width:20%; height: auto; margin-top: -5vh;" />
+  </div>
+
+  <svg
+    class="h-auto -mt-6 w-6/12 absolute right-0 top-0 z-[-1]"
+    style="min-width:400px"
+    xmlns="http://www.w3.org/2000/svg"
+    preserveAspectRatio="none"
+    viewBox="0 0 800 800">
+    <ellipse
+      cx="700"
+      cy="100"
+      rx="600"
+      ry="600"
+      fill="currentColor"
+      style="color:#c5deff; opacity:0.4" />
+    <ellipse
+      cx="700"
+      cy="100"
+      rx="520"
+      ry="550"
+      fill="currentColor"
+      style="color:#c5deff; opacity:0.7" />
+    <ellipse
+      cx="700"
+      cy="100"
+      rx="450"
+      ry="500"
+      fill="currentColor"
+      style="color:#c5deff" />
+  </svg>
   <slot />
 </template>
 
 <script setup lang="ts">
+
 </script>
+
+<style scoped>
+.logo-container {
+  position: relative;
+  z-index: 1;
+  width: 100%;
+  margin: 0 auto;
+  text-align: center;
+display: flex;
```

Comment:
```suggestion
  display: flex;
```

Esto lo debería de coger el linter, pero por alguna razón no lo hace, sorry 😅